test/regression.uts

```diff
@@ -513,12 +513,16 @@ assert all(p.hashret() == p2.hashret() for p in px)
 assert not any(p.answers(p2) for p in px)
 assert all(p2.answers(p) for p in px)
 conf.checkIPinIP = conf_back
+
+a1, a2 = Net("www.google.com"), Net("www.secdev.org")
 prt1, prt2 = 12345, 54321
 s1, s2 = 2767216324, 3845532842
 p1 = IP(src=a1, dst=a2)/TCP(flags='SA', seq=s1, ack=s2, sport=prt1, dport=prt2)
 p2 = IP(src=a2, dst=a1)/TCP(flags='R', seq=s2, ack=0, sport=prt2, dport=prt1)
 assert p2.answers(p1)
 assert not p1.answers(p2)
+# Not available yet because of IPv6
```

Comment:
Is'nt it better to tag this test with ipv6 and disable them in Travis and AppVeyor ?

Reply:
Done